src/lib/apis/cart/get-logged-cart.api.ts

```
@@ -0,0 +1,51 @@
+        // Parse response
+        const payload = (await res.json()) as CartSuccessResponse | CartErrorResponse;
+
+        // Check if the request was successful
+        return {
+            ok: res.ok,
+            status: res.status,
+            payload,
+        };
+    } catch (error) {
+        // Log and re-throw the error
+        console.error("Error fetching cart data:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)